Blish HUD/Controls/Dropdown.cs

```diff
@@ -7,13 +7,16 @@
 using MonoGame.Extended.TextureAtlases;
 
 namespace Blish_HUD.Controls {
-    public class Dropdown:Control {
+    /// <summary>
+    /// Represents a Guild Wars 2 Dropdown control.
+    /// </summary>
+    public class Dropdown : Control {
 
-        private class DropdownPanel:Control {
+        private class DropdownPanel : Control {
 
             private Dropdown _assocDropdown;
 
-            protected int _highlightedItem = -1;
+            private int _highlightedItem = -1;
             private int HighlightedItem {
```

Comment:
Private field with getter and setter? Is this because SetProperty() invalidates the layout?